src/status_im/chat/screen.cljs

```diff
@@ -100,7 +120,8 @@
                                 (when (not= height layout-height)
                                   (re-frame/dispatch [:set-layout-height height]))))}
      [chat-toolbar]
-     [messages-view group-chat]
+     (when (= :chat current-view)
+       [messages-view group-chat])
```

Comment:
messages-view is called with group-chat, but interface changed to take two args. Am I missing something?

Reply:
looks like rebase leftover, will fix interface